spec/requests/admin/reports_controller_spec.rb

```diff
@@ -75,4 +75,20 @@
       expect(response.body).to match(/My suggested improvement/)
     end
   end
+
+  describe "GET application digest" do
+    subject(:get_request) { get admin_application_digest_csv_path(format: :csv) }
+
+    before { create(:application_digest) }
+
+    it "renders successfully" do
+      get_request
+      expect(response).to have_http_status(:ok)
+    end
+
+    it "sends the data" do
+      get_request
+      expect(response.body).to match(/^legal_aid_application_id,firm_name,provider_username,date_started,date_submitted,days_to_submission,use_ccms,matter_types,proceedings,passported,df_used,earliest_df_date,df_reported_date,working_days_to_report_df,working_days_to_submit_df,employed,hmrc_data_used,referred_to_caseworker,true_layer_path,bank_statements_path,true_layer_data,has_partner,contrary_interest,partner_dwp_challenge,applicant_age,non_means_tested,family_linked,family_linked_lead_or_associated,number_of_family_linked_applications,legal_linked,legal_linked_lead_or_associated,number_of_legal_linked_applications,no_fixed_address,biological_parent,parental_responsibility_agreement,parental_responsibility_court_order,child_subject,parental_responsibility_evidence,autogranted,ecct_routed/)
```

Comment:
could we extend to test some of the content as well as the headers. You can use compound matchers

```ruby
expect(response.body)
  .to match(/this/)
  .and match(/that/)
```